roles/node/tasks/download_binary.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Set system variables
+  set_fact:
```

Comment:
lets add create tmp too since we are reading it from tmp_dir variable, if we change that it must exist